packages/manager/modules/pci/src/projects/project/storages/object-storage/users/add/add.routing.js

```diff
@@ -20,7 +20,7 @@ export default /* @ngInject */ ($stateProvider) => {
         ) =>
           PciStoragesUsersService.getUsersCredentials(
             projectId,
-            allUserList.map((user) => user.id),
+            allUserList.filter((user) => user != null),
```

Comment:
```suggestion
            allUserList.filter((user) => user),
```

Reply:
Okay Anoop taken care all the suggestions.